### DISCUSSION TITLE
Does Telegraf support Layer 133 API changes?

### DISCUSSION THREAD

**Discussion by ghost** [telegraf/telegraf#1562] [Q&A]
Hi team, 

According to https://core.telegram.org/api/layers, we need to switch to 64-bit identifiers for users, groups, supergroups and channels.

Does Telegraf already have this implemented? If so, starting with which release? 

Thanks!

**Comment by NormandoHall** [comment]
Only ensure your dB can store this 64 bit integer (in case you use a dB). Telegraf already support 64-bit identifiers.
